From b035451b5666d991b10d99d5c84cc2a015853e9a Mon Sep 17 00:00:00 2001
From: Shivani Agarwal <shivania2@vmware.com>
Date: Mon, 10 Jun 2024 09:50:47 +0000
Subject: [PATCH] systemd denial fix (rebased for 43.6)

AVC avc:  denied  { audit_control } for  pid=1 comm="systemd" capability=30
scontext=system_u:system_r:init_t:s0 tcontext=system_u:system_r:init_t:s0
tclass=capability permissive=0

Signed-off-by: Shivani Agarwal <shivani.agarwal@broadcom.com>
---
 policy/modules/system/init.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/modules/system/init.te b/policy/modules/system/init.te
--- a/policy/modules/system/init.te
+++ b/policy/modules/system/init.te
@@ -174,7 +174,7 @@
 #
 
 # Use capabilities. old rule:
-allow init_t self:capability ~{ audit_control audit_write sys_module };
+allow init_t self:capability ~{ audit_write sys_module };
 allow init_t self:capability2 ~{ mac_admin mac_override };
 allow init_t self:cap_userns all_cap_userns_perms;
 allow init_t self:netlink_netfilter_socket create_socket_perms;
